### PR TITLE
added all the proto files

### PR DESCRIPTION
**Pull request by dtracers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # ReplayAnalysis
 Various tools for decompiling / analyzing Rocket League replays
+
+# setup
```
Comment (Sciguymjm) [inline README.md]: Consistency with capitalization (same with Windows below)

File: api/metadata/player_loadout.proto [hunk 2]
```diff
@@ -0,0 +1,15 @@
+syntax = "proto2";
+
+package api.metadata;
+
+message PlayerLoadout {
+    optional int32 loadOutId = 1;
```
Comment (Sciguymjm) [inline api/metadata/player_loadout.proto]: Is this supposed to be camel case?

**Comment by twobackfromtheend** [comment]
I've looked through the structures of the created classes, but I don't feel up to looking through all the refactored hit/stats code.
I think I'm just gonna trust you on this one, and if there are specific parts you'd like me to look over, do let me know - I will be able to look through smaller bits, but it coming as a whole is really hard to read :/ The state of the code wasn't great to begin with (before you started refactoring/rearrange/reformatting) but at least it was mine so I sorta knew where everything lay and what I had in mind when writing it. 

tl;dr: I need more practice reading code but this is a whole lot of changes to what was quite messy code, and I just don't feel like following everything through, but do let me know of bits you've changed that are more high-risk at introducing bugs.

File: replay_analysis/analysis/stats/stats_manager.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+from ...generated.api.stats.team_stats_pb2 import TeamStats
+from ...generated.api.team_pb2 import Team
+from ...json_parser.game import Game
+
+
+class StatsManager:
+
+    def get_stats(self, game: Game, proto_game: game_pb2.Game, player_map: Dict[str, Player],
+                  data_frames):
+        self.calculate_player_stats(game, proto_game, player_map, data_frames)
```
Comment (twobackfromtheend) [inline replay_analysis/analysis/stats/stats_manager.py]: Would be good to specify this order in the image - important as as the stats calculated before can't depend on those calculated after.
Also rearrange the function definitions in this file to follow the same order :P

File: import_fixer.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                new_file.write(line)
+    if modified:
+        os.remove(file_path)
+        shutil.move(abs_path, file_path)
+        print(str(len(replace_map)), 'imports')
+    else:
+        print('not modified')
+
+
+def prevent_leaks(top_level_dir='generated', exclude_dir=None, top_level_import="api"):
```
Comment (kcolton) [inline import_fixer.py]: @dtracers Is there a danger of creating leaks if protobuffs are imported the wrong way? 

Not gonna try to understand this file, but just curious going by the file and function name if there are any new landmines unique to protobuf usage that need to be known.

File: proto.sh [hunk 5]
```diff
@@ -0,0 +1,11 @@
+echo "Creating proto files"
+./replay_analysis/generated/protoc --proto_path=. --python_out=./replay_analysis/generated ./**/*.proto
+./replay_analysis/generated/protoc --proto_path=. --python_out=./replay_analysis/generated ./api/**/*.proto
```
Comment (kcolton) [inline proto.sh]: @dtracers Doesn't `**` perform arbitrarily deep matching? Wouldn't `./api/**/*.proto` always be found by `./**/*.proto`?

**Comment by kcolton** [comment]
May also be worth noting that in addition to protobufs, this also adds hdf5 usage, especially as it's another platform dependency.
